pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -462,6 +462,3 @@ def pre_dispatch(self, trainer: "pl.Trainer") -> None:
 
     def dispatch(self, trainer: "pl.Trainer") -> None:
         """Hook to do something at trainer run_stage starts."""
-
-    def post_dispatch(self, trainer: "pl.Trainer") -> None:
```

Comment:
This hook is used by https://github.com/ray-project/ray_lightning and thus (indirectly) also by https://github.com/intel-analytics/BigDL . Should we call it within teardown and raise a warning (if implemented)?